### PR TITLE
Global Colors: New Global Color Modal - Made the color name duplication notice dynamic.

### PR DESCRIPTION
**Pull request by udidol**

### CODE REVIEW

**Review by matipojo — DISMISSED** [review]

File: core/kits/assets/js/globals/global-select-behavior.js [hunk 1]
```diff
@@ -268,12 +268,34 @@ export default class GlobalControlSelect extends Marionette.Behavior {
 			onShow: () => {
 				// Put focus on the naming input.
 				this.ui.globalNameInput = this.confirmNewGlobalModal.getElements( 'widget' ).find( 'input' ).focus();
+				this.ui.confirmMessageText = this.confirmNewGlobalModal.getElements( 'widget' ).find( '.e-global__confirm-message-text' );
```
Comment (kobizz) [inline core/kits/assets/js/globals/global-select-behavior.js:271]: Insert `this.confirmNewGlobalModal.getElements( 'widget' )` into a local variable

File: core/kits/assets/js/globals/global-select-behavior.js [hunk 2]
```diff
@@ -268,12 +268,34 @@ export default class GlobalControlSelect extends Marionette.Behavior {
...
+
+				this.ui.globalNameInput.on( 'change paste keyup', () => this.onAddGlobalConfirmInputChange() );
```
Comment (kobizz) [inline core/kits/assets/js/globals/global-select-behavior.js:273]:
```suggestion
				this.ui.globalNameInput.on( 'input', () => this.onAddGlobalConfirmInputChange() );
```

File: core/kits/assets/js/globals/global-select-behavior.js [hunk 3]
```diff
@@ -268,12 +268,34 @@ export default class GlobalControlSelect extends Marionette.Behavior {
...
 	}
 
+	onAddGlobalConfirmInputChange() {
+		if ( ! this.view.globalsList ) {
+			return;
+		}
+
+		let messageContent;
+
+		for ( const globalColor of Object.values( this.view.globalsList ) ) {
```
Comment (kobizz) [inline core/kits/assets/js/globals/global-select-behavior.js:287]:
```suggestion
		for ( const globalValue of Object.values( this.view.globalsList ) ) {
```

File: core/kits/assets/js/globals/global-select-behavior.js [hunk 4]
```diff
@@ -268,12 +268,34 @@ export default class GlobalControlSelect extends Marionette.Behavior {
...
+			if ( this.ui.globalNameInput.val() === globalColor.title ) {
+				messageContent = '<i class="eicon-info-circle"></i> ' + elementor.translate( 'global_color_name_already_exists' );
+				break;
+			} else {
+				messageContent = elementor.translate( 'global_color_confirm_text' );
```
Comment (kobizz) [inline core/kits/assets/js/globals/global-select-behavior.js:292]: Use view functions

**Review by kobizz — CHANGES_REQUESTED** [review]

**Review by kobizz — APPROVED** [review]

**Review by matipojo — APPROVED** [review]

**Review by KingYes — APPROVED** [review]

**Pull request merged**